tailwind.config.js

```diff
@@ -16,12 +16,28 @@ module.exports = {
         'post': '0.01rem'
       },
       fontFamily: {
-        'inter': ['Inter', 'sans-serif']
+        'inter': ['Inter', 'sans-serif'],
+        'lato': ['Lato', 'sans-serif']
       },
       colors: {
+        gray02: '#E0E0E0',
+        gray05: '#F5F5F5',
         gray06: '#828282',
         grayIcon: '#C4C4C4',
-        primaryBlue: '#2f80ed'
+        foundationBlue: '#0D5AA0',
+        primaryBlue: '#2f80ed',
+        humanId_blue: '#023B60',
+        cyan: '#00ADB5',
+        cyan10: '#00ADD5'
+      },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color names are not very descriptive. It's recommended to use more meaningful names for colors. For example, instead of `gray02`, `gray05`, and `humanId_blue`, you could use `lightGray`, `veryLightGray`, and `darkBlue` respectively. This will make it easier for other developers to understand what each color represents.

```diff
-        gray02: '#E0E0E0',
-        gray05: '#F5F5F5',
-        humanId_blue: '#023B60',
+        lightGray: '#E0E0E0',
+        veryLightGray: '#F5F5F5',
+        darkBlue: '#023B60',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->